docs/components/collapse.md

```diff
@@ -97,7 +97,7 @@ If no label is provided, only a caret will be displayed.
 
 KCollapse can be controlled with `v-model`.
 
-<div class="collapse-margin-bottom">
+<div class="collapse-parent-wrapper">
```

Comment:
Can we not use the word `parent` in class name? `collapse-wrapper` is perfect

Reply:
fixed